configtx/application.go

```diff
@@ -176,10 +177,21 @@ func (a *ApplicationGroup) Policies() (map[string]Policy, error) {
 	return getPolicies(a.applicationGroup.Policies)
 }
 
+// SetModPolicy sets the specified modification policy for the application group
+func (a *ApplicationGroup) SetModPolicy(modPolicy string) error {
+	if modPolicy == "" {
+		return fmt.Errorf("failed to set empty mod policy to application group")
```

Comment:
The error messages don't really reflect the issue. The error that's being raised should indicate that a modification policy is required; we didn't fail to set it, the caller didn't provide it.

Additionally, when there are no fillins, we should be using `errors.New` instead of `fmt.Errorf`.

These comments apply to all of the errors that were introduced that indicate a modification policy was not provided.